examples/src/main/scala/io/chrisdavenport/shellfish/Place.scala

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
It's my duty to ask you to do the most boring thing of computer science: "Can you add this in documentation too?" :D